src/main/scala/uk/gov/hmrc/ArtifactoryConnector.scala

```diff
@@ -23,25 +23,27 @@ import dispatch.Defaults._
 import dispatch._
 import play.api.libs.json.Json
 import play.api.libs.json.Json.toJsFieldJsValueWrapper
-import sbt.DirectCredentials
+import sbt.{DirectCredentials, Logger}
 
 class ArtifactoryConnector(httpClient: Http, credentials: DirectCredentials, repositoryName: String) {
   private val targetRepository = "bintray-distribution"
 
-  def deleteVersion(artifact: ArtifactDescription): Future[String] = {
+  def deleteVersion(artifact: ArtifactDescription, logger: Logger): Future[String] = {
```

Comment:
just return `Future[Unit]` to keep in line with `BintrayConnector.delete` functions? Otherwise just looks like it will result in double logging since it is logging the message as well as returning.

Reply:
There are some tests that are asserting on this error string. They're a bit pointless so I'll just delete them. :)